packages/testsuite/cypress/support/form-editing.ts

```diff
@@ -163,10 +163,19 @@ Cypress.Commands.add("clearListAttributeItems", (attributeName) => {
     });
 });
 
-Cypress.Commands.add("selectText", (formId, attributeName, value) => {
-  cy.formInput(formId, attributeName).select(value, { force: true });
-  cy.formInput(formId, attributeName).should("have.value", value);
-  cy.formInput(formId, attributeName).trigger("change", { force: true });
+Cypress.Commands.add("selectInDropdownMenuOnPage", (elementId, value) => {
+  // somtimes can be the dropdown menu behind a pop-up alert notification. So we need close them.
+  cy.closeAllPopUpNotifications();
+  cy.get(`button[data-id="${elementId}"]`).click();
+  cy.get(`button[data-id="${elementId}"]`)
+    .parent()
+    .within(() => {
+      cy.get(`a.dropdown-item`).contains(value).click();
+    });
+});
+
+Cypress.Commands.add("selectInDropdownMenuInForm", (formId, fieldName, value) => {
+  cy.selectInDropdownMenuOnPage(`${formId}-${fieldName}-editing`, value);
```

Comment:
`cy.formInput(formId, attributeName)` should be used instead of `${formId}-${fieldName}-editing`

Reply:
formInput returns the form input not a drop down menu input. 
see the difference: 
> cy.get("#" + configurationFormId + "-" + attributeName + "-editing");

vs
> cy.get(`button[data-id="${elementId}"]`)

the `elementId` is `${formId}-${fieldName}-editing`

Reply:
OK, I missed that.